src/components/ADempiere/Form/VPOS2/Options/salesOrder.vue

```diff
@@ -570,6 +570,7 @@ export default defineComponent({
       },
       // setter
       set(show) {
+        console.log({ show })
```

Comment:
please remove this console.log

Reply:
remove console    https://github.com/solop-develop/frontend-core/pull/1804/commits/1197d3806a38ed49944926e0d1762c2b4c7c2afe